app/models/concerns/form526_claim_fast_tracking_concern.rb

```diff
@@ -336,5 +337,44 @@ def log_flashes
   rescue => e
     Rails.logger.error("Failed to log Flash Prototypes #{e.message}.", backtrace: e.backtrace)
   end
+
+  def log_document_type_metrics
+    return if in_progress_form.blank?
+
+    fd = in_progress_form.form_data
+    fd = JSON.parse(fd) if fd.is_a?(String)
+    additional_docs_by_type = get_doc_type_counts(fd.fetch('additionalDocuments', fd.fetch('additional_documents', [])))
+    private_medical_docs_by_type = get_doc_type_counts(
+      fd.fetch('privateMedicalRecordAttachments', fd.fetch('private_medical_record_attachments', []))
+    )
+
+    log_document_type_metrics_for_group(additional_docs_by_type, 'additional_documents')
+    log_document_type_metrics_for_group(private_medical_docs_by_type, 'private_medical_record_attachments')
+
```

Comment:
`return if additional_doc_counts.blank? && private_doc_counts.blank?`

Could you add this here? 

Then you could remove the `if additional_docs_by_type.present? || private_medical_docs_by_type.present?` below